src/Athens-Examples/AthensSurfaceExamples.class.st

```diff
@@ -549,7 +550,8 @@ AthensSurfaceExamples class >> exampleDrawForm [
 
 		can pathTransform translateX: 30 Y: 30. 
 		can draw ].
-	Display getCanvas drawImage: surf asForm  at: 0@0
+	
+	surf asForm asMorph openInWindow
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?